deployments/hadoop-yarn/ansible/39-create-user-scripts.yml

```diff
@@ -42,45 +46,127 @@
     create_hdfs_user: |
             #!/bin/bash
             # Setup HDFS user directory
-            NEW_USERNAME=$1
+            NEW_USERNAME=${1:?}
+            NEW_USERTYPE=${2:?}
+            NEW_HDFSROOT=${3:?}
+            NEW_HDFS_PATH=/$NEW_HDFSROOT/$NEW_USERNAME
```

Comment:
Not sure if this change works. Or we may be missing a configuration in HDFS to make it work.
When testing this version I get an HDFS "Permission Denied"error message. If I then manually create the hdfs directory: "/user/$NEW_USERNAME" Spark executes the cell without an error

Reply:
This part works, but I'm passing in the wrong prefix for NEW_HDFS_ROOT from the script that calls this.
I'll fix it and add a new commit.